src/sdl/sdl_touch.c

```diff
@@ -138,7 +160,12 @@ void _al_sdl_touch_input_event(SDL_Event *e)
       return;
    }
 
-   int touchId = e->tfinger.fingerId;
+   int touchId = find_touch_state_index_with_id(e->tfinger.fingerId);
```

Comment:
Maybe this variable should be called something else to reduce confusion? `touch_idx`?

Reply:
Makes sense, updated.